code/drasil-gool/lib/GOOL/Drasil/LanguageRenderer/CommonPseudoOO.hs

```diff
@@ -442,3 +443,34 @@ inherit n = toCode $ maybe empty ((colon <+>) . text) n
 
 implements :: (Monad r) => [Label] -> r ParentSpec
 implements is = toCode $ colon <+> text (intercalate listSep is)
+
+-- Swift and Julia --
+boolRender :: String
+boolRender = "Bool"
+
+bool :: (RenderSym r) => VSType r
+bool = typeFromData Boolean boolRender (text boolRender)
+
+docMod' :: (RenderSym r) => String -> String -> [String] -> String -> SFile r -> SFile r
+docMod' = docMod modDoc'
+
+-- | Generates a doc comment for a language that isn't supported by Doxygen.
+-- Since Doxygen doesn't support the language, it uses dashes instead of backslashes.
+modDoc' :: ModuleDocRenderer
+modDoc' desc as date m = m : [desc | not (null desc)] ++
+      [docField authorDoc (stringList as) | not (null as)] ++
+      [docField dateDoc date | not (null date)] ++
+      [docField noteDoc watermark]
+
+-- | Creates an arbitrary DocC Markup field for documentation.
```

Comment:
DocC is a swift-specific term (see https://www.swift.org/documentation/docc/). It's based on Markdown, so it may still be a good candidate as a language-agnostic alternative to doxygen, but this comment specifically referencing DocC should probably be updated now that this function is being used for more than just swift.

If Julia has its own standard syntax for documentation generation, it would be good for GOOL to generate that syntax (perhaps a future improvement?)

Reply:
Oh, thanks for pointing that out @bmaclach!  I'll make sure to edit the comment about DocC, and see if there are any better ways to treat doc comments in Julia.

Reply:
I looked at documentation for Julia.  It has a [built-in documentation system](https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation) where you put a string with markdown-formatted documentation above an object, and it automatically assigns that information to that object.  Since it uses markdown, it should work well to use the same format as the DocC format for Julia.